action-api/src/main/java/com/powsybl/action/TerminalsConnectionAction.java

```diff
@@ -81,4 +84,31 @@ public Optional<ThreeSides> getSide() {
     public boolean isOpen() {
         return open;
     }
+
+    @Override
+    public NetworkModification toModification() {
+        // not obvious how to map on ConnectableConnection/PlannedDisconnection/... especially if there is a side selected
+        return getSide().map(value -> new TerminalsConnectionModification(getElementId(), value, isOpen()))
+                .orElse(new TerminalsConnectionModification(getElementId(), isOpen()));
```

Comment:
Instead of using a new NetworkModification, it would be better to use the existing `ConnectableConnection` and `PlannedDisconnection`/`UnplannedDisconnection`.
It would just need to add an optional `side` parameter in those NetworkModifications and in the following `AbstractConnectable.connect/disconnect` methods and add the potential filter on `side` in the `for` loop in `AbstractConnectable.connect()` (and disconnect).

Reply:
Thanks, I am trying!